docs/reference/Image.rst

```diff
@@ -49,7 +49,14 @@ Functions
 
 .. autofunction:: open
 
-    .. warning:: To protect against potential DOS attacks caused by "`decompression bombs`_" (i.e. malicious files which decompress into a huge amount of data and are designed to crash or cause disruption by using up a lot of memory), Pillow will issue a `DecompressionBombWarning` if the image is over a certain limit. If desired, the warning can be turned into an error with `warnings.simplefilter('error', Image.DecompressionBombWarning)` or suppressed entirely with `warnings.simplefilter('ignore', Image.DecompressionBombWarning)`. See also `the logging documentation`_ to have warnings output to the logging facility instead of stderr.
+    .. warning::
+        To protect against potential DOS attacks caused by "`decompression bombs`_" (i.e. malicious files
+        which decompress into a huge amount of data and are designed to crash or cause disruption by using up
+        a lot of memory), Pillow will issue a `DecompressionBombWarning` if the image is over a certain
```

Comment:
Not sure whether this should be a straight up code block or possibly a :class: reference